### PR TITLE
design and internet slow

### PR DESCRIPTION
**Pull request by srikanthgudipudi**
desgin and handle issues

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "keycloak-angular": "^4.0.2",
     "keycloak-js": "^4.6.0",
     "ngx-mat-select-search": "^1.8.0",
-    "ngx-spinner": "^6.1.2",
```
Comment (kiranharidas187) [inline package.json:39]: Why are we using an external plugin for spinner

File: src/app/modules/users/add-single-user/add-single-user.component.html [hunk 2]
```diff
@@ -12,5 +12,11 @@ <h5 class="title">Add User</h5>
 </mat-dialog-content>
 <mat-dialog-actions class="custom-dialog-actions" align="end">
   <button mat-button mat-dialog-close>CANCEL</button>
-  <button mat-button style="background: #bababa;" (click)="onSubmit()" cdkFocusInitial>ADD USER</button>
+  <button [disabled]="submitClick" style="background: #bababa;" color = 'primary' (click)="onSubmit()" mat-button>
```
Comment (kiranharidas187) [inline src/app/modules/users/add-single-user/add-single-user.component.html]: Please make a component and resuse it

File: src/styles.scss [hunk 3]
```diff
@@ -2633,10 +2633,10 @@ th.mat-sort-header-sorted {
   color: $primary-color;
 }
 
-// .mat-table {
-//   overflow: auto;
-//   height: 550px;
-// }
+.mat-table {
+  overflow: auto;
+  height: 550px;
```
Comment (kiranharidas187) [inline src/styles.scss]: Why is this hardcoding of height

File: package.json (hunk 1 above)
Comment (srikanthgudipudi) [reply package.json:39]: It is used in sl-portal-library

File: src/styles.scss (hunk 3 above)
Comment (srikanthgudipudi) [reply src/styles.scss]: To fix the height of the table Even few records or no records

File: src/app/admin-footer/admin-footer.component.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<div class="footer">
```
Comment (kiranharidas187) [inline src/app/admin-footer/admin-footer.component.html]: Is should be inside admin-core

File: src/styles.scss [hunk 5]
```diff
@@ -2655,21 +2790,30 @@ th.mat-sort-header-sorted {
...
+.mat-row {
+  height: 48px;
+}
+.mat-spinner {
+  ::ng-deep circle {
+    stroke: $white-color !important;
+  }
+}
 
+html {
```
Comment (kiranharidas187) [inline src/styles.scss:2809]: Why is this styles?

Comment (srikanthgudipudi) [reply src/styles.scss:2809]: To disable the side nav bar when modal open

**Pull request merged**